lib/ngx/errlog.lua

```diff
@@ -50,39 +51,51 @@ function _M.set_filter_level(level)
 end
 
 
-function _M.get_logs(max, logs)
+function _M.get_logs(max, logs, opts)
+    local is_fetch_time = opts and opts.fetch_time
```

Comment:
I think we can simply return the UNIX time unconditionally. This Lua API has never included in any releases, it's fine for us to drop backward compatibility here. (Well, even if we release this code, we can still change the API without notice since this API is marked experimental in the docs).